### PR TITLE
[ONNX] Refactor quantized ops tests

### PR DESCRIPTION
**Pull request by postrational**

### CODE REVIEW

File: test/util/test_case.hpp [hunk 1]
```diff
@@ -75,11 +90,34 @@ namespace ngraph
...
-                    function_output_type, function_output_shape, values));
+                    function_output_type, expected_shape, values));
 
                 ++m_output_index;
             }
 
+            template <typename T>
+            void add_expected_output(const std::vector<T>& values)
+            {
+                auto shape = m_function->get_results().at(m_output_index)->get_shape();
```
Comment (tsocha) [inline test/util/test_case.hpp]: What if output shape is broken?
Default shape as 1D tensor can be better in this case.
Now expected shape is connected with ngraph which can be invalid.

File: test/util/test_case.hpp [hunk 2]
```diff
@@ -75,11 +90,34 @@ namespace ngraph
...
+            void add_expected_output_from_file(ngraph::Shape expected_shape,
+                                               const std::string& basepath,
+                                               const std::string& filename)
+            {
+                auto filepath = ngraph::file_util::path_join(basepath, filename);
+                add_expected_output_from_file<T>(expected_shape, filepath);
+            }
+
+            template <typename T>
+            void add_expected_output_from_file(ngraph::Shape expected_shape,
```
Comment (tsocha) [inline test/util/test_case.hpp]: Why there is no signature for expected output from file without passing expected_shape?
Maybe we should differ function by types not names?

Comment (postrational) [reply test/util/test_case.hpp]: The signature without shape is just a backward-compatible workaround. In general, we should be providing expected shapes.

File: test/util/test_case.hpp (hunk 1 above)
Comment (postrational) [reply test/util/test_case.hpp]: We cannot assume that the output shape will be flat.

**Pull request merged**